src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -167,7 +167,7 @@ public static String wrap(String in, int wrapAmount) {
 
         String[] lines = in.split("\n");
         StringBuilder result = new StringBuilder();
-        addWrappedLine(result, lines[0], wrapAmount);
+        addWrappedLine(result, lines[0].replaceAll("\\s+$", ""), wrapAmount);
```

Comment:
The RegEx is `CharMatcher.WHITESPACE.trimTrailingFrom` (see http://www.leveluplunch.com/java/examples/right-trim-string/).